File: tests/fixtures/golden/typescript/visibilityPrivateOwner.ts
class SecretHelper {
  // Private: only SecretHelper's own members may call it directly. #75
  // fixture (paired with visibilityPrivateProber.ts): a cross-file
  // two-segment fallback call to this must stay UNRESOLVED.
  private static doSecret(): string {
    return "secret";
  }

  static callLocally(): string {
    return SecretHelper.doSecret();
  }
}

File: tests/fixtures/golden/rust/src/visibility_private_owner.rs
/// Private item: only `visibility_private_owner` may call it directly by
/// name. #75 fixture (paired with `visibility_private_prober`): a
/// cross-module bare call to this must stay `UNRESOLVED`, not bind
/// through the name fallback.
fn secret_helper() -> &'static str {
    "secret"
}

pub fn call_locally() -> &'static str {
    secret_helper()
}
